This patch solves an issue with depdemo tests when there is no
backslash safe echo/print command available.
--- libtool-2.4.6/tests/depdemo.at.ORIG	2016-08-10 09:33:40 +0200
+++ libtool-2.4.6/tests/depdemo.at	2016-08-10 09:34:27 +0200
@@ -202,7 +202,7 @@
       $ECHO     '  int i;'
       $ECHO     '  for (i = 0; i < indent; i++)'
       $ECHO     "    putchar (' ');"
-      $ECHO     '  printf ("l'$i' (%i)\n", var_l'$i');'
+      $ECHO     '  printf ("l'$i' (%i)\\n", var_l'$i');'
       if test 1 -eq $i; then
         $ECHO   '  var_l1++;'
       else
@@ -227,11 +227,11 @@
       $ECHO     '  int i;'
       $ECHO     '  for (i = 0; i < indent; i++)'
       $ECHO     "    putchar (' ');"
-      $ECHO     '  printf ("l'$i' (%i)\n", var_l'$i');'
+      $ECHO     '  printf ("l'$i' (%i)\\n", var_l'$i');'
       $ECHO     '  func_l'$j'(indent+1);'
       $ECHO     '  for (i = 0; i < indent; i++)'
       $ECHO     "    putchar (' ');"
-      $ECHO     '  printf("libm [cos (0.0) = %g]\n", (double) cos ((double) 0.0));'
+      $ECHO     '  printf("libm [cos (0.0) = %g]\\n", (double) cos ((double) 0.0));'
       $ECHO     '  var_l'$i' += var_l'$j';'
       $ECHO     '  return 0;'
       $ECHO     '}'
